docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -921,10 +848,102 @@ Highest number of active threads in the thread pool.
 `thread_pool.<thread_pool_name>.completed`::
 (integer)
 Number of tasks completed by the thread pool executor.
+====
+
+[[cluster-nodes-stats-api-response-body-fs]]
+===== `fs` section
```

Comment:
Note to reviewers:

There is still some work that needs to be done in this section (adding data type, adding a few missing parms). However, I decided to address those as part of a separate PR rather than bloat this one.